apps/distgo/cmd/docker/build_test.go

```diff
@@ -35,6 +35,7 @@ import (
 	"github.com/palantir/godel/apps/distgo/cmd/dist"
 	"github.com/palantir/godel/apps/distgo/cmd/docker"
 	"github.com/palantir/godel/apps/distgo/params"
+	dockerparams "github.com/palantir/godel/apps/distgo/params/docker"
```

Comment:
Why is this file moved to a subdirectory? I'm always a bit skeptical about introducing a package that is immediately aliased